models/pancreas_ct_dints_segmentation/docs/README.md

```diff
@@ -53,6 +52,16 @@ Three channels
 - Label 1: pancreas
 - Label 0: everything else
 
+### GPU Memory Consumption
+
+Dataset Class: CacheDataset
+Data Size: 420 3D Volumes
+Cache Rate: 1.0
+Multi-GPU VRAM: (at least) 16GB GPUs
```

Comment:
Hi @dongyang0122 , should here be RAM rather than GPU memory?

Reply:
I too experienced confusion in this regard. I included the GPU memory requirement in this section based on the provided description.

Reply:
Ah, sorry, @yiheng-wang-nv pointed out in the other MR that it defaults to System RAM rather than VRAM

Reply:
@zephyrie Thanks!

@yiheng-wang-nv Could you kindly provide an example for my reference?